games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
I know that you were the previous maintainer, i followed every step of the minetest package since its very beginning in your own public overlay over its import to the portage tree till now when i got the (proxy-)maintainer.

It is already agreed on that this ebuild is out of the games herd responsibility as you can read for example at the first comment by @mgorny in my last PR #17.